src/index.html

```diff
@@ -2,11 +2,495 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width,
+      initial-scale=1.0"
+    >
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+    <link
+      rel="stylesheet"
+      href="./styles/main.scss"
+    >
+
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo">
+        <img
+          src="images/logo.png"
+          class="logo"
+          alt="logo"
+        >
+      </a>
+      <nav class="navigation">
+        <ul class="navigation__ul">
+          <li class="navigation__item">
+            <a
+              class="navigation__link is-active"
+              href="#Apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              class="navigation__link"
+              href="#Samsung"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              class="navigation__link"
+              href="#Smartphones"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="navigation__item">
+            <a
+              class="navigation__link"
+              data-qa="nav-hover"
+              href="#Laptops"
+            >
+                Laptops & computers
+            </a>
```

Comment:
Check correct indents for the content
```suggestion
            >
              Laptops & computers
            </a>
```